packages/documentation-framework/components/example/example.js

```diff
@@ -173,12 +177,36 @@ export const Example = ({
     + (loc.pathname.endsWith(source) ? '' : `/${source}`)
     + '/'
     + slugger(title);
-    
+
   return (
     <div className="ws-example">
       <div className="ws-example-header">
         <AutoLinkHeader
-          metaText={isBeta && <Badge className="ws-beta-badge pf-v5-u-ml-xs">Beta</Badge>}
+          metaText={
+            <React.Fragment>
+              {isBeta && (
+                <Tooltip content="This beta component is currently under review and is still open for further evolution.">
+                  <Button variant="plain">
+                    <Label isCompact color="blue">Beta</Label>
+                  </Button>
+                </Tooltip>
+              )}
+              {isDemo && (
+                <Tooltip content="Demos show how multiple components can be used in a single design.">
+                  <Button variant="plain">
+                    <Label isCompact color="purple">Demo</Label>
+                  </Button>
+                </Tooltip>
+              )}
+              {isDeprecated && (
+                <Tooltip content="Deprecated components are available for use but are no longer being maintained or enhanced.">
```

Comment:
Maybe a Surge issue not updating from a previous commit, but I'm seeing slightly different tooltip messages for these three items (Beta, Demo, Deprecated)

![Screenshot 2023-05-24 at 4 55 25 PM](https://github.com/patternfly/patternfly-org/assets/8651509/42845a18-ef15-4a9a-93d7-a330363c7985)

Reply:
good catch! i'll fix that

Reply:
Sorry - the text in my screenshot above actually maps to the [tooltip in mdx.js](https://github.com/patternfly/patternfly-org/pull/3547/files#diff-da135efaa2b01f501f1ac196874e6c6ae745345196f59a4581445639bf5384c4R253), but curious if this text should match or maybe live in a single variable somewhere that can be referenced?

Reply:
I made them match. We don't right now have like a constants or strings file. Maybe that's something else we can clean up to add when we clean up the docs-framework in the future.